src/netconf_confirmed_commit.c

```diff
@@ -884,6 +891,10 @@ np2srv_confirmed_commit_cb(sr_session_ctx_t *session, const struct lyd_node *inp
         /* there is already a pending confirmed commit, keep its backup, but the timeout will be reset */
         timer_delete(commit_ctx.timer);
         commit_ctx.timer = 0;
+
+        /* send notification about extending timeout for confirmed-commits */
+        np_send_notif_confirmed_commit(nc_sess, session, NP_CC_EXTEND, timeout, 0);
```

Comment:
Just to make the code a bit more readable I would just set the flag `timeout_changed` here and then later, based on that, send one or the other notification.